dusk-network/dusk-blockchain
# Automate race detection - enable race option on build and devnet.sh

# New features

* enable race option on devnet.sh
* enable race option on build.go
* create new task build-race on Makefile

# How to test
Run `make build-race` and observe that the install cmd is executed with `-race` flag
Run `./devnet.sh -r true` and observe that the install cmd is executed with `-race` flag

closes #588
